person.rb

```diff
@@ -20,6 +21,10 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals << rental
+  end
+
```

Comment:
- [ ] This is a many-to-many relationship which should be set up a little differently than a has-many / belongs-to relationship, as you can see in [this example](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) you should create a new instance of Rental and pass the date, person and the book as arguments. (make sure to notice how the person is passed)